src/components/DocHeadMotif.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { PureComponent } from 'react';
```

Comment:
Looks like you have expanded `@databyss-org/ui/modules/Landing/Entries` into the DocHead component. Instead, make your changes in the `@databyss-org/ui` repo.